functions/import_summary_data.R

```diff
@@ -0,0 +1,49 @@
+#' @title Import and clean School Summary Statistics data
+#' 
+#' @description This function reads in the desired sheet of the Summary
+#' Statistics dataset and cleans variable names.
+#'
+#' @param sheet_name Name of data sheet to import
+#' @param calendar_year Year of dataset to import in calendar year format; 
+#' e.g. 2020.
+#'
+#' @return Tibble of cleaned data for given dataset and year.
+#' 
+#' @export
+#'
+#' @examples
+#' import_summary_data("LA and SCOT", 2021)
+#' import_summary_data("LA and SCOT", "timeseries")
+
+import_summary_data <- function(sheet_name, calendar_year) {
+  
+  # Define pipe
```

Comment:
I would need clearer explanation of what this does and I don't know :)

Reply:
Is this better?

Reply:
Not sure if you can see it in the wee code preview but you should be able to see it if you go to the same line in Files Changed tab